besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -2261,7 +2261,9 @@ private EthNetworkConfig updateNetworkConfig(final NetworkName network) {
 
     if (bootNodes != null) {
       try {
-
+        if (!peerDiscoveryEnabled) {
```

Comment:
We could move this block out of the try ...

Reply:
done